src/com/untamedears/realisticbiomes/RealisticBiomes.java

```diff
@@ -332,15 +342,14 @@ private void registerEvents() {
 			pm.registerEvents(new GrowListener(this), this);
 			pm.registerEvents(new SpawnListener(materialGrowth, fishSpawn), this);
 			pm.registerEvents(new PlayerListener(this, materialGrowth), this);
+			pm.registerEvents(new ClickableInventoryListener(), this);

```

Comment:
I know the docu says you should register the listener again if you dont extend ACivMod, but thats actually wrong and will break the whole thing (I guess I should update that).